backends/p4tools/modules/testgen/targets/bmv2/test/P4Tests.cmake

```diff
@@ -72,8 +72,14 @@ if(P4TOOLS_TESTGEN_BMV2_TEST_PTF)
       "BMv2 PTF tests are enabled, but the Python3 module 'google.rpc' can not be found. BMv2 PTF tests will fail."
     )
   endif()
+  # Filter some programs  because they have issues that are not captured with Xfails.
+  set (P4C_V1_TEST_SUITES_P416_PTF ${P4C_V1_TEST_SUITES_P416})
+  list(REMOVE_ITEM P4C_V1_TEST_SUITES_P416_PTF
+       # A particular test (or packet?) combination leads to an infinite loop in the simple switch.
+       "${P4C_SOURCE_DIR}/testdata/p4_16_samples/v1model-special-ops-bmv2.p4"
```

Comment:
There is a bug in either PTF or the simple switch that cause it too loop packets infinitely under certain conditions. For now, I am disabling this test, but this is something we could look into.